src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1075,11 +1075,10 @@ private static NamedTypeSymbol GetDelegateType(BoundExpression expr)
                 methods = constructedMethods.ToImmutableAndFree();
             }
 
-            if (methods.Length == 1 && !extensionMethodsOfSameViabilityAreAvailable)
+            if (methods.Length == 1 && methods[0].Arity == 0 && !extensionMethodsOfSameViabilityAreAvailable)
```

Comment:
Could you comment on the case of explicitly provided type arguments?

---
In reply to: [67027377](https://github.com/dotnet/roslyn/pull/11981#discussion_r67027377) [](ancestors = 67027377,67019197)